docs/authentication/social-connections/oauth.mdx

```diff
@@ -106,18 +106,18 @@ export async function GET() {
 
 ## Add social connection after sign-up
 
-For each OAuth provider, you can choose whether it will be available during sign-up and sign-in, or if the connection should be made later.
+For each OAuth provider, you can disable the option to sign up and sign in to your application using the provider. This is especially useful for users that want to connect their OAuth account after authentication.
 
-This is especially useful for applications that prefer to connect third-parties after the fact. For example, a Github connection can be made after sign-up if an application wants to read repository data.
+For example, say your application wants to read a user's GitHub repository data but doesn't want to allow the user to authenticate with their GitHub account. The user can sign up with their email and password, or whatever method you choose, and then afterwards, connect their GitHub account to your application.
```

Comment:
There's still a screenshot on line 38 of this page. Intentional?

Reply:
![image](https://github.com/clerk/clerk-docs/assets/5156915/5ae04def-7ab5-4e19-ac02-65a7f656806c)

Reply:
let me see what I can do about the copy to see if we can indicate it better

Reply:
good point ! [2cfad17](https://github.com/clerk/clerk-docs/pull/1071/commits/2cfad17a6d45bf493126dc1cb722b727995ee826)